app/controllers/settings_controller.rb

```diff
@@ -6,6 +6,10 @@ def change_locale
     lang = I18n.default_locale unless I18n.available_locales.include?(lang)
     cookies.permanent[:plots2_locale] = lang
     I18n.locale = lang
-    redirect_to request.referer || root_url
+    if request.referer
+      redirect_to request.referer+ "?_=" + Time.now.to_i.to_s[-4..-1] 
```

Comment:
can you add an extra space before the `+`,  like `request.referer+ "?_=" + `... ? Very small thing, I know, but it's very helpful to have exact coding practices over the long term. Thank you !